pep-0604.rst

```diff
@@ -73,13 +74,59 @@ Here are some examples of what we can do with this feature.
     assert str | int == Union[str,int]
     assert str | int | float == Union[str, int, float]
 
-    assert isinstance("", int | str)
-    assert issubclass(bool, int | float)
+    # Instead of
+    # typing.List[typing.Union[str, int]]
+    typing.List[str | int]
+
+    # list[typing.Union[str, int]] (Currently TypeError: 'type' object has no attribute '__getitem__')
```

Comment:
I don't think that error is relevant. PEP 585 makes it work starting in 3.9. You might merge this with the previous example like you did for dict/Dict below.